### PR TITLE
Fix: Align items controls are not working (...) [ED-8382]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

File: assets/dev/scss/frontend/widgets/spacer.scss [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 	& > .elementor-widget-spacer {
 		width: var( --container-widget-width, var( --spacer-size ) );
-		align-self: stretch;
-		flex-shrink: 0;
+		--align-self: var( --container-widget-align-self, initial );
+		--flex-shrink: 0;
```
Comment (hein-obox) [inline assets/dev/scss/frontend/widgets/spacer.scss:18]: I have limited the usage of `--align-self: stretch` to the 'container row`. The `container column` will use the `initial` value.

File: assets/dev/scss/frontend/widgets/spacer.scss [hunk 2]
```diff
@@ -29,4 +29,8 @@
 			}
 		}
 	}
+
+	&.elementor-widget-empty .elementor-widget-container {
+		position: absolute;
```
Comment (hein-obox) [inline assets/dev/scss/frontend/widgets/spacer.scss:34]: I have fixed the display of the empty placeholder.

**Review by TzviRabinovitch — CHANGES_REQUESTED** [review]
please fix conflicts

**Comment by hein-obox** [comment]
@TzviRabinovitch Merge conflicts solved.

**Pull request merged**